src/main/java/net/glowstone/entity/passive/GlowAbstractHorse.java

```diff
@@ -35,6 +36,19 @@
             .put(Material.HAY_BLOCK, TickUtil.minutesToTicks(3))
             .build();
 
+    @Override
+    protected boolean tryFeed(Material food, GlowPlayer player) {
+        if (isAdult() && !isTamed()) {
```

Comment:
Remove a nesting level by inverting this if-statement.

Reply:
Done.